assets/src/scss/pages/_listing-page.scss

```diff
@@ -0,0 +1,153 @@
+@mixin render-bullet {
+  &:before {
+    display: block;
+    margin: 0 $sp-1;
+    content: "•";
+  }
+}
+
+.query-list-item {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  margin-top: var(--query-list--item--margin-top, $sp-4);
+
+  @include medium-and-up {
+    flex-direction: row;
+  }
+}
+
+.query-list-item-image {
+  flex-basis: 30%;
+
+  &.query-list-item-image-max-width {
+    img {
+      width: 100%;
+      height: 310px;
+      object-fit: cover;
+      object-position: 50% 50%;
+      margin-right: $sp-2;
+
+      html[dir="rtl"] & {
+        margin-right: 0;
+        margin-left: $sp-2;
+      }
+
+      @include medium-and-up {
+        height: 200px;
+      }
+
+      @include large-and-up {
+        height: 250px;
+      }
+
+      @include x-large-and-up {
+        height: 260px;
+      }
+
+      @include xx-large-and-up {
+        height: 240px;
+      }
+    }
+  }
+
+  .wp-post-image {
+    height: 100%;
+  }
+}
+
+.query-list-item-body {
+  flex-basis: 70%;
+  flex-shrink: 0;
+  margin-top: $sp-1x;
+
+  header {
+    margin-top: $sp-1;
+  }
+
+  // Chrome and Safari
+  // :has is not fully support for now.
+  // https://developer.mozilla.org/en-US/docs/Web/CSS/:has
+  &:has(.taxonomy-action-type):has(.taxonomy-post_tag) {
+    .wrapper-post-tag {
+      display: flex;
+      @include render-bullet;
+    }
+  }
+}
+
+.query-list-item-post-terms {
+  display: flex;
+  font-size: $font-size-sm;
+  font-family: $roboto;
+
+  // Firefox
```

Comment:
This piece of code will be removed after the `:has` pseudo class is being fully supported by Firefox